RELEASE/scripts/autoscend/paths/wildfire.ash

```diff
@@ -235,20 +235,26 @@ boolean LX_wildfire_pump(int target)
 	while(target > my_wildfire_water() && get_counters("", 0, 0) == "")
 	{
 		//r25706. clicking in browser works properly. but visit url causes a desync on water quantity. and there is no mafia command to pump water while keeping water level synced. As such we must visit charpane.php after pumping water to update our water value.
+		int start_adv = my_adventures();
 		visit_url("place.php?whichplace=wildfire_camp&action=wildfire_oldpump");
 		visit_url("charpane.php");
+		if(start_adv == my_adventures())
+		{
+			abort("Tried to pump water but our adv count did not change. what went wrong?");
+		}
 		if(start_water == my_wildfire_water())
 		{
 			abort("Mafia failed to update your water level after pumping water");
 		}
 		boolean completely_full = stomach_left() < 1 && inebriety_left() < 1;
-		if(my_adventures() <= auto_advToReserve() && completely_full)
+		int adv_target = auto_advToReserve();
+		if(!completely_full)
 		{
-			break;	//we are done for the day
+			adv_target++;
```

Comment:
What is purpose of this? Why reserve an extra adventure if  we aren't full?

Reply:
To avoid prematurely triggering some things that happen when we are down to out 0 reserved adv left.
Normally we eat or drink whenever we drop down below 10 adv remaining. this is more aggressive and goes down to reserve+1. (typically 2 or 3 adv remaining)

Reply:
i am trying to avoid prematurely triggering reserve = 0 stuff prematurely when not full.
conversly when we are full I want it to actually go down to 0 reserved remaining to make sure they do trigger properly